src/settings/enums/Layout.ts

```diff
@@ -15,14 +15,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import PropTypes from 'prop-types';
-
 /* TODO: This should be later reworked into something more generic */
 export enum Layout {
     IRC = "irc",
     Group = "group",
     Bubble = "bubble",
 }
-
-/* We need this because multiple components are still using JavaScript */
-export const LayoutPropType = PropTypes.oneOf(Object.values(Layout));
```

Comment:
Did you mean to include this?

Reply:
Yes, its a maintenance task that seems a little silly to have in its own PR. Bubbles related enough

Reply:
:+1: